tutorial/markdown/aspnet/quickstart-csharp-aspnet/quickstart-csharp-aspnet.md

```diff
@@ -3,16 +3,16 @@
 path: "/tutorial-quickstart-csharp-aspnet"
 title: Quickstart in Couchbase with C# and ASP.NET 
 short_title: ASP.NET and C# Start
-description: 
-  - Build a REST API with Couchbase's C# SDK and ASP.NET
-  - Learn how to connect to a cluster to create, read, update, and delete documents
-  - Write simple parameterized N1QL queries
+description:
+  - Learn to build a REST API with Couchbase's C# SDK 3.4 and ASP.NET
```

Comment:
Do we need the SDK 3.4 specifically?

Reply:
Currently the latest one is the 3.4 series. We can change it later when we have 3.5 series.